t.carp

```diff
@@ -0,0 +1,6 @@
+(defmodule Foo
```

Comment:
Should this file be part of the PR?

Reply:
Also some muck from my old commits, geez-- I stepped these back via another commit, but it looks like I should have squashed and amended, oof.

Pulling master into this branch should fix this.

Reply:
No, it must come from @scolsen commit that got into my branch. File removed.